deploifai/mlflow/get_setup.py

```diff
@@ -0,0 +1,93 @@
+import click
+import pyperclip
+
+from deploifai.api import DeploifaiAPIError
+from deploifai.context import pass_deploifai_context_obj, DeploifaiContextObj
+from PyInquirer import prompt
+
+
+@click.command('get-setup')
+@click.option("--external", is_flag=True, help="Use code externally")
+@pass_deploifai_context_obj
+def get_setup(context: DeploifaiContextObj, external):
+    """
+    Command to obtain code for mlflow integration
+    """
+
+    # assume that the user should be in a project directory, that contains local configuration file
+    if 'id' in context.local_config['PROJECT']:
+        project_id = context.local_config['PROJECT']['id']
+        # getting all the required information to provide the code
+        fragment = """
+                    fragment project on Project {
+                        name
+                        account{
+                            username
+                        }
+                        experiments{
+                          name
+                          environment
+                          resourceAccessTokenString
+                        }
+                        
+                    }
+                    """
+        project_data = context.api.get_project(
+            project_id=project_id, fragment=fragment
+        )
+        project_name = project_data["name"]
+        workspace_name = project_data["account"]["username"]
+        experiment_data = project_data["experiment"]
+        experiment_names = [experiment["name"] for experiment in experiment_data]
+        experiment_environments = [experiment["environment"] for experiment in experiment_data]
+        experiment_tokens = [experiment["resourceAccessTokenString"] for experiment in experiment_data]
+
+        # selecting experiment name from the options
+        if len(experiment_names) == 0:
+            click.secho("No experiments exist for the project")
+        # elif len(experiment_names) == 1:
+        # experiment_name = experiment_names[0]
+        else:
+            choose_experiment = prompt(
+                {
+                    "type": "list",
+                    "name": "experiment",
+                    "message": "Choose a experiment",
+                    "choices": [
+                        exp for exp in experiment_names
```

Comment:
There should be a better way to write this. Look at [cloud profile prompt](https://github.com/deploifai/cli/blob/077a9c612c33f15b6087e9ffc784234c674ad95d/deploifai/project/create.py#L113) from the project create command.

Reply:
That is where I got the idea for the prompt
What do you think should be changed to make it better?